integrationTests/chainSimulator/mempool/testutils_test.go

```diff
@@ -353,8 +356,18 @@ func testOnProposed(t *testing.T, sw *core.StopWatch, numTxs int, numAddresses i
 	require.Equal(t, numTxs, len(selectedTransactions))
 
 	proposedBlock1 := createProposedBlock(selectedTransactions)
-	sw.Start(t.Name())
 
+	f := createPprofFiles(t)
```

Comment:
Now I see that this function does not belong to the common test utilities. I think it belongs in the file that implements the benchmarks.